src/data/developer-docs-links.yaml

```
@@ -10,31 +10,42 @@
   items:
     - id: docs-nav-intro-to-ethereum
       to: /developers/docs/intro-to-ethereum/
+      descp_id: docs-nav-intro-to-ethereum-descp
```

I think we should change `descp` to `description` everywhere. It's more readable, explicit and in line with [what we are doing elsewhere](https://github.com/ethereum/ethereum-org-website/blob/76f5b87e225fcb379474fce2942898b3a342f46b/src/intl/en/page-wallets-find-wallet.json#L31). 

```suggestion
      description: docs-nav-intro-to-ethereum-description
```